index.bs

```diff
@@ -625,6 +637,20 @@ Documents in [=prerendering browsing contexts=] are [=Document/hidden=].
 
 [=Prerendering browsing contexts=] never have [=system focus=].
 
+<h3 id="performance-navigation-timing-extension">Extensions to the {{PerformanceNavigationTiming}} interface</h3>
+
+Extend the {{PerformanceNavigationTiming}} interface as follows:
+
+<pre class="idl">
+  partial interface PerformanceNavigationTiming {
+      readonly attribute DOMHighResTimeStamp activationStart;
+  };
+</pre>
+
+When getting the value of the <dfn attribute for="PerformanceNavigationTiming">activationStart</dfn> attribute, run the following steps:
+
+1. Return a {{DOMHighResTimeStamp}} with a time value equal to the [=current document=]'s [=Document/activation start time=].
```

Comment:
Might be worthwhile to add a note here saying that this value is 0 in cross-origin prerenders.

Reply:
Done